open_dread_rando/door_patcher.py

```diff
@@ -383,6 +389,8 @@ def rename_shields(self, door: Container, scenario: str):
             # update the minimap entry as well
             mapBlockages = self.editor.get_scenario_map(scenario).raw.Root.mapBlockages
             mapBlockages[new_id] = copy.deepcopy(mapBlockages[old_sName])
+            if link_name.startswith("wpRight"):
+                mapBlockages[new_id]["bFlipX"] = True
```

Comment:
worth adding a comment here mentioning that this is done so we can optimize the icons file